src/sonic-host-services/.gitignore

```diff
@@ -1,4 +1,5 @@
 # Compiled Python files
+.pytest_cache/
```

Comment:
This belongs in the `# Unit test coverage` section if we're going to keep it. Please see my suggestion.

Reply:
OK